client/a8c-for-agencies/sections/agency-tier/lib/get-tier-benefits.ts

```diff
@@ -0,0 +1,93 @@
+import { preventWidows } from 'calypso/lib/formatting';
+
+interface Benefit {
+	title: string;
+	description: string;
+	features: string[];
+	isComingSoon: boolean;
+	availableTiers: ( 'emerging-partner' | 'agency-partner' | 'pro-agency-partner' )[];
+}
+
+const getTierBenefits = ( translate: ( key: string ) => string ): Benefit[] => [
+	{
+		title: translate( 'Sales Training & Resources' ),
+		description: translate(
+			'Foundational product training, marketing materials, and advanced sales training.'
+		),
+		features: [
+			translate( 'Pro partners eligible to receive advanced sales training upon request.' ),
+		],
+		isComingSoon: true,
+		availableTiers: [ 'emerging-partner', 'agency-partner', 'pro-agency-partner' ],
+	},
+	{
+		title: translate( 'Support' ),
+		description: preventWidows(
+			translate( 'Program support, priority product support*, and dedicated partner managers*.' )
+		),
+		features: [
+			translate(
+				'Pro partners receive access to assigned partner manager & priority support across Automattic products.'
+			),
+		],
+		isComingSoon: true,
+		availableTiers: [ 'emerging-partner', 'agency-partner', 'pro-agency-partner' ],
+	},
+	{
+		title: translate( 'Earning & Growth Opportunities' ),
+		description: translate(
+			'Reseller and referral earning opportunities, access to leads generated through agency directory listing(s)*, and pre-qualified leads from Automattic sales teams*.'
+		),
+		features: [
+			translate(
+				'Agency Partners eligible to receive leads generated through agency directory listings.'
+			),
+			translate(
+				'Pro Partners eligible to receive pre-qualified leads from Automattic sales teams when opportunities arise as well as leads generated through agency directory listings.'
+			),
+		],
+		isComingSoon: true,
+		availableTiers: [ 'emerging-partner', 'agency-partner', 'pro-agency-partner' ],
+	},
+	{
+		title: translate( 'Networking & Community' ),
+		description: translate(
+			'Access to Automattic community platforms and advisory board* for strategic insights.'
+		),
+		features: [
+			translate( 'Pro partners eligible to receive invitation to join Automattic Advisory Board.' ),
+		],
+		isComingSoon: true,
+		availableTiers: [ 'emerging-partner', 'agency-partner', 'pro-agency-partner' ],
+	},
+	{
+		title: translate( 'Tools & Platforms' ),
+		description: translate(
+			'Agency dash for streamlined client billing, program and product management, and free agency site on WordPress.com or Pressable*.'
+		),
+		features: [
+			translate( 'Agency partners receive free WordPress.com and Pressable site.' ),
```

Comment:
Should the word "site" be plural here?

Reply:
That's a good catch. I think it's better to keep this `Agency partners receive a free WordPress.com site and a Pressable site.` Let me confirm this with the design team and fix it. Thanks!

Reply:
I have updated the text here: https://github.com/Automattic/wp-calypso/pull/95548

Thanks!